src/containers/Tenants/Tenants.tsx

```diff
@@ -218,12 +299,16 @@ export const Tenants = ({additionalTenantsProps}: TenantsProps) => {
     };
 
     return (
-        <TableWithControlsLayout>
-            <TableWithControlsLayout.Controls>{renderControls()}</TableWithControlsLayout.Controls>
-            {error ? <ResponseError error={error} /> : null}
-            <TableWithControlsLayout.Table loading={loading}>
-                {currentData ? renderTable() : null}
-            </TableWithControlsLayout.Table>
-        </TableWithControlsLayout>
+        <div className={b('table-wrapper')}>
+            <TableWithControlsLayout>
```

Comment:
is it possible to pass className directly to TableWithControlsLayout as a prop?

Reply:
TableWithControlLayout has `min-width: 100%`, parent container here have almost full screen width, I need to limit it to place button in the right corner of a table, not the right cornet of the screen

Without wrapper: 
<img width="2292" alt="Screenshot 2025-04-03 at 15 19 05" src="https://github.com/user-attachments/assets/5f883d8c-9e66-41dd-92d0-1faaf56484d4" />

With wrapper:
<img width="2304" alt="Screenshot 2025-04-03 at 15 19 12" src="https://github.com/user-attachments/assets/4dcce638-6fca-460d-8915-3f0f95dc251d" />